cmd/trade.go

```diff
@@ -506,7 +506,19 @@ func runTradeCmd(options inputs) {
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
-	userID := "-1" // TODO DS Properly generate and save user ID.
+	var userIDPrehash string
+	if botConfig.IsTradingSdex() {
+		userIDPrehash = botConfig.TradingAccount()
+	} else {
+		userIDPrehash = amplitudeAPIKey
+	}
+
+	userIDHashed, e := utils.HashString(userIDPrehash)
```

Comment:
can you add a comment describing why we take the hash?
reason is so we don't expose the user's account or api_key.